name-generator.py

```diff
@@ -0,0 +1,43 @@
+import random
+
+# Function to randomly select a specified number of names from a list of names
```

Comment:
Good use of comments: The code contains useful comments that explain what each part of the code does. This makes it easier for others to understand and maintain the code.